src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -84,7 +85,20 @@ export default class ProcessConverter extends AbstractConverter<Process> {
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private buildFlowNodeBpmnElements(processId: string, bpmnElements: Array<any> | any, kind: ShapeBpmnElementKind): void {
-    ensureIsArray(bpmnElements).forEach(bpmnElement => convertedFlowNodeBpmnElements.push(new ShapeBpmnElement(bpmnElement.id, bpmnElement.name, kind, processId)));
+    ensureIsArray(bpmnElements).forEach(bpmnElement => {
```

Comment:
:information_source: I initially planed the following refactoring here to avoid calling `convertedFlowNodeBpmnElements.push` twice.
I finally didn't push it as IMHO, it doesn't bring more clarity, and won't probably won't fit the implementation when managing new start event type like terminate end event.

```typescript
private buildFlowNodeBpmnElements(processId: string, bpmnElements: Array<any> | any, kind: ShapeBpmnElementKind): void {
    ensureIsArray(bpmnElements).forEach(bpmnElement => {
        let addBpmnElement = true;
        if (kind == ShapeBpmnElementKind.EVENT_START) {
            // get the list of eventDefinition hold by the Start Event bpmElement
            const eventDefinitions = Object.values(EventDefinition).filter(eventDefinition => {
                return bpmnElement.hasOwnProperty(eventDefinition);
            });
            // None Start Event only if we do not have any EventDefinition
            if (eventDefinitions.length != 0) {
                addBpmnElement = false;
            }
        }

        if (addBpmnElement) {
            convertedFlowNodeBpmnElements.push(new ShapeBpmnElement(bpmnElement.id, bpmnElement.name, kind, processId));
        }
    });
}
```

Reply:
I think we could refactor it when there will be more events ;)